### ISSUE TITLE
Проблема в логике

### ISSUE THREAD

**Issue by uzarsalan** [callmesveta/Frontend-developer-course#1]
Является ли блюдо вегетарианским, если в нем есть только один вегетарианский ингредиент?

https://github.com/callmesveta/Frontend-developer-course/blob/97c21d8ef9a50c9ee9ad08175ced3deb60d53845/script.js#L48

**Comment by callmesveta** [comment]
Да, точно, должно быть "every" вместо "some" для определения всех вегетарианских ингредиентов в блюде)
